### ISSUE TITLE
BUG CMemoryBlock floats

### ISSUE THREAD

**Issue by kanersps** [Freeeaky/GTALua#46]
It returns these weird values
http://puu.sh/hT6Oi/64207110db.png
When using this code:
http://puu.sh/hTcB8/199632c30f.png

**Comment by Freeeaky** [comment]
Yep, can confirm there is something wrong. It seems that memory is in tact, but the returned value is somehow wrong. I'll fix this later.
Thanks for taking the time to post it :)

**Comment by MockbaTheBorg** [comment]
The code is almost correct, but the right offsets to obtain the XYZ values are 0, 2 and 4, not 0, 8 and 16.
I think this can be marked as Closed.
